app/views/kudos/show.html.erb

```diff
@@ -21,6 +21,8 @@
   <% @kudo.receiver_id %>
   <%= @kudo.receiver_email %>
 </p>
-
-<%= link_to 'Edit', edit_kudo_path(@kudo) %> |
+<% if @kudo.giver_email == current_employee.email %>
```

Comment:
maybe `kudo.giver_of_kudo == current_employee` ?

Reply:
that's what i needed, but i wanted to do this quick and i make something that worked and did not fixed it later